modules/testsuite/cxf-tests/src/test/java/org/jboss/test/ws/jaxws/cxf/clientConfig/SSLContextElytronClientConfigTestCase.java

```diff
@@ -0,0 +1,122 @@
+              .addClass(org.jboss.test.ws.jaxws.cxf.httpauth.HelloResponse.class)
+              .addClass(org.jboss.test.ws.jaxws.cxf.httpauth.ObjectFactory.class)
+              .addAsWebInfResource(new File(JBossWSTestHelper.getTestResourcesDir() + "/jaxws/cxf/httpauth/WEB-INF/wsdl/helloMutualSSLport.wsdl"), "wsdl/hello.wsdl");
+      return archive;
+   }
+
+   @Before
+   public void startContainerAndDeploy() throws Exception {
+      if (!containerController.isStarted(TWO_WAY_SSL_CONTAINER)) {
+         containerController.start(TWO_WAY_SSL_CONTAINER);
+         try {
+            deployer.deploy(DEPLOYMENT);
+         } catch (Exception e) {
+            System.err.println(e);
+         }
+      }
+   }
+
+   @Test
+   @RunAsClient
+   public void testConfiguredSSLContext() throws Exception {
+      AuthenticationContext previousAuthContext = AuthenticationContext.getContextManager().getGlobalDefault();
+      try {
+         ElytronClientTestUtils.setElytronClientConfig(JBossWSTestHelper.getTestResourcesDir() + "/jaxws/cxf/clientConfig/META-INF/wildfly-config-ssl-context.xml");
+         QName serviceName = new QName("http://jboss.org/http/security", "HelloService");
+         URL wsdlURL = getResourceURL("jaxws/cxf/httpauth/WEB-INF/wsdl/hello.wsdl");
+         Service service = Service.create(wsdlURL, serviceName);
+         Hello proxy = service.getPort(Hello.class);
+         ((BindingProvider) proxy).getRequestContext().put(BindingProvider.ENDPOINT_ADDRESS_PROPERTY, "https://localhost:13443/ssl-mutual-auth/HelloService");
+         CXFClientConfigurer cxfClientConfigurer = new CXFClientConfigurer();
+         cxfClientConfigurer.setConfigProperties(proxy, null, null);
+         assertEquals(100, proxy.helloRequest("number"));
+      } finally {
+         AuthenticationContext.getContextManager().setGlobalDefault(previousAuthContext);
+      }
+   }
+
+   @Test
+   @RunAsClient
+   public void testNotConfiguredSSLContext() throws Exception {
+      AuthenticationContext previousAuthContext = AuthenticationContext.getContextManager().getGlobalDefault();
+      SSLContext previousDefaultSSLContext = SSLContext.getDefault();
+      try {
+         SSLContext emptySSLContext = SSLContext.getInstance("TLS");
+         emptySSLContext.init(null, new TrustManager[] { null }, null);
+         // for this test we must remove default SSL context because it accepts existing certificate without additional configuration
+         SSLContext.setDefault(emptySSLContext);
+         QName serviceName = new QName("http://jboss.org/http/security", "HelloService");
+         URL wsdlURL = getResourceURL("jaxws/cxf/httpauth/WEB-INF/wsdl/hello.wsdl");
+         Service service = Service.create(wsdlURL, serviceName);
+         Hello proxy = service.getPort(Hello.class);
+         ((BindingProvider) proxy).getRequestContext().put(BindingProvider.ENDPOINT_ADDRESS_PROPERTY, "https://localhost:13443/ssl-mutual-auth/HelloService");
```

Comment:
could you please parametrize 13443 so that it takes port offset for tested container into account? Eg. https://github.com/jbossws/jbossws-cxf/blob/master/modules/testsuite/cxf-tests/src/test/java/org/jboss/test/ws/jaxws/samples/wsse/policy/oasis/WSSecurityPolicyExamples23xTestCase.java#L68-L69

Reply:
Fixed.